fvm/src/kernel/default.rs

```diff
@@ -558,6 +578,8 @@ where
         h2: &[u8],
         extra: &[u8],
     ) -> Result<Option<ConsensusFault>> {
+        self.call_manager
+            .charge_gas(self.call_manager.price_list().on_extern_traversal())?;
```

Comment:
@raulk we should discuss this. The fact that this traverses an extern is an implementation detail and the actual _cost_ of that depends on the implementation. We should bundle the cost into the verify consensus fault syscall itself.